.github/workflows/publish.yml

```diff
@@ -111,12 +156,12 @@ jobs:
         run: |
           git config --local user.name 'reactorbot'
           git config --local user.email '32325210+reactorbot@users.noreply.github.com'
-          git tag -m "Release milestone ${{ needs.prepare.outputs.fullVersion }}" v${{ needs.prepare.outputs.fullVersion }} ${{ github.sha }}
+          git tag -m "Release milestone ${{ needs.reactor-netty-core.outputs.fullVersion }}" v${{ needs.reactor-netty-core.outputs.fullVersion }} ${{ github.sha }}
           git push --tags
 
   tagRelease:
     name: Tag release
-    needs: [ prepare, deployRelease ]
+    needs: [ reactor-netty-core, reactor-netty-http, reactor-netty-http-brave, reactor-netty-incubator-quic, deployRelease ]
```

Comment:
same comment here as in `tagMilestone`. limiting the number of needed jobs will also avoid a lot of extra connecting lines in the UI btw 😄

Reply:
fixed